docs/swagger_v3/channels.spec.yaml

```diff
@@ -55,7 +69,7 @@ schemas:
         description: The last height in which the channel was updated on chain
         example: 141956
       last_updated_tx_type:
-        type: string
+        $ref: '#/components/schemas/ChannelLastUpdatedTxType'
         description: The transaction type of the last transaction that updated the channel
```

Comment:
https://swagger.io/docs/specification/v3_0/using-ref/#ref-and-sibling-elements
Either inline enum definition or move `description` to `ChannelLastUpdatedTxType`